tareekh.go

```diff
@@ -66,8 +66,8 @@ func ToShortDate(t time.Time) string {
 
 // FromDateString parses date string of format YYYY-MM-DD and returns time object
 func FromDateString(dt string) (time.Time, error) {
-	localizedFormat := fmt.Sprintf("%s %s", DefaultDateFormat, "MST")
 	zone, _ := Now().Zone()
+	localizedFormat := fmt.Sprintf("%s %s", DefaultDateFormat, zone)
```

Comment:
https://golang.org/pkg/time/#pkg-constants 
zone should be "MST" in the format string.. not the actual zone itself. 

The original code looks ok to me. please change it back to what it was.

Reply:
Unit test failed from here, because here there is CET timezone and the time has -1 hour and does not match with the given one in the test. I can change it back if doesn't matter